source/renderer/app/api/api.js

```diff
@@ -964,16 +966,26 @@ export default class AdaApi {
         LOVELACES_PER_ADA
       );
 
+      let delegationDepositSum = 0;
```

Comment:
@tomothespian please change this to "totalDeposits" and push it above the line where you calculate `transactionFee`: https://github.com/input-output-hk/daedalus/blob/728ec6e0978bb4fd34e21d61fb29abc925768ba5/source/renderer/app/api/api.js#L965-L967

Then you will have a nicer way to calculate the fee:
```
const fee = new BigNumber(totalInputs - totalOutputs - totalDeposits).dividedBy(
    LOVELACES_PER_ADA
);
```